src/components/OrderableList/OrderableList.stories.tsx

```diff
@@ -0,0 +1,142 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { Checkbox, CheckboxState } from "@components/Checkbox/Checkbox";
+import { useListData } from "@react-stately/data";
+import { ItemDropTarget } from "@react-types/shared";
+import { Meta, Story } from "@storybook/react";
+import React, { ReactElement, useState } from "react";
+import { Button } from "@components/Button/Button";
+import { OrderableList } from "./OrderableList";
+import { merge } from "@utilities/merge";
+import { GridNode } from "@react-types/grid";
+import { DragProperties, ItemDragState, OrderableListItem } from "./types";
+
+const renderContent = (
+    { value, prevKey, nextKey }: GridNode<OrderableListItem>,
+    { componentDragState, isFocusVisible }: DragProperties,
+) => {
+    return (
+        <div
+            className={merge([
+                "tw-break-word tw-border tw-border-solid tw-rounded tw-p-3",
+                componentDragState === ItemDragState.Dragging && "tw-bg-black-10 tw-border-black-20 tw-opacity-75",
```

Comment:
You could create a Record in the `types.ts`, e.g.:
```
export const dragStateStyles: Record<ItemDragState, string> = {
    [ItemDragState.Dragging]: 'tw-bg-black-10 tw-border-black-20 tw-opacity-75',
    [ItemDragState.Idle]: 'tw-border-black-20',
    [ItemDragState.Preview]: 'tw-bg-white tw-border-violet-70 tw-border-4',
};
```
and then apply the styles like so:
```
className={merge([..., dragStateStyles[componentDragState], ...])}
```

Reply:
done